python/tvm/runtime/profiler_vm.py

```diff
@@ -35,10 +37,17 @@ class VirtualMachineProfiler(vm.VirtualMachine):
 
     def __init__(self, exe, device, memory_cfg=None):
         super(VirtualMachineProfiler, self).__init__(exe, device, memory_cfg)
-        self.module = _ffi_api._VirtualMachineDebug(exe.module)
+
+        # Make sure the constructor of the VM module is on the proper device
+        if device.device_type >= rpc_base.RPC_SESS_MASK:
```

Comment:
Checking device type using a `>=` is kind of confusing. If there isn't a better way to check can you explain why the `>=` works in a comment.

Reply:
done